client/src/components/Navigation/Navigation.js

```diff
@@ -129,6 +129,7 @@ const Navigation = () => {
                   <Route path="/explore" component={Explore} />
                   <Route exact path="/alg-sets" component={AlgSetList} />
                   <Route exact path="/alg-sets/:id" component={AlgSet} />
+                  <Route exact path="/alg-sets/:id/:name" component={AlgSet} />
```

Comment:
We should indicate that the `:name` part is optional,  so that id-only URLs still work `path="/alg-sets/:id/:name?"`.

Reply:
I tried doing this and I think it wasn't working.

Reply:
Interesting, I remember trying that and seemed to behave as expected. Gonna check again later.

Reply:
Indeed, it does work for me (a single route instead of two different ones)
```jsx
<Route exact path="/alg-sets/:id/:name?" component={AlgSet} />
```